webviz_ert/controllers/plot_view_controller.py

```diff
@@ -13,6 +13,14 @@
 from webviz_ert.plugins._webviz_ert import WebvizErtPluginABC
 
 
+def _suffix_to_plot_type(suffix: str) -> str:
+    if suffix == "resp":
+        return "response"
+    if suffix == "param":
+        return "parameter"
+    raise Exception(f"Unexpected suffix `{suffix}`")
```

Comment:
which part? the exception? or the function? it makes the code easier to read, I feel, to extract this in a bit of own logic, we can test it (hurr hurr), and easily extend it and use it elsewhere... just feels tidier to me, but maybe it's just my feelings and it's premature optimization, or something like that. i can leave that block (without the exception) in the function that calls this one, too, just feel like it's a nice little stand alone thing that can be pulled out (for great success).
what downsides am i not seeing, because i am often blind for them once i'm convinced that it should be like this or that 😅?

Reply:
removed in 59b97b9